docs/release-notes/template.rst

```diff
@@ -0,0 +1,4 @@
+:Type: Kickstart / TUI / GUI / Modularization / ...
+:Summary: Summarize the change (#bug number)
+:Description: Describe the change in detail.
```

Comment:
I would show in this example how to do multiline description.

Reply:
Fixed.